src/olympia/lib/settings_base.py

```diff
@@ -153,21 +153,62 @@ def cors_endpoint_overrides(internal, public):
 # Note: If you update this list, don't forget to also update the locale
 # permissions in the database.
 AMO_LANGUAGES = (
-    'af', 'ar', 'bg', 'bn-BD', 'ca', 'cs', 'da', 'de', 'dsb',
-    'el', 'en-GB', 'en-US', 'es', 'eu', 'fa', 'fi', 'fr', 'ga-IE', 'he', 'hu',
-    'hsb', 'id', 'it', 'ja', 'ka', 'kab', 'ko', 'nn-NO', 'mk', 'mn', 'nl',
-    'pl', 'pt-BR', 'pt-PT', 'ro', 'ru', 'sk', 'sl', 'sq', 'sv-SE', 'uk', 'ur',
-    'vi', 'zh-CN', 'zh-TW',
+    'af',  # Afrikaans
+    'ar',  # Arabic
+    'bg',  # Bulgarian
+    'bn-BD',  # Bengali (Bangladesh)
+    'ca',  # Catalan
+    'cs',  # Czech
+    'da',  # Danish
+    'de',  # German
+    'dsb',  # Lower Sorbian
+    'el',  # Greek
+    'en-GB',  # English (British)
+    'en-US',  # English (US)
+    'es',  # Spanish
+    'eu',  # Basque
+    'fa',  # Persian
+    'fi',  # Finnish
+    'fr',  # French
+    'ga-IE',  # Irish
+    'he',  # Hebrew
+    'hsb',  # Upper Sorbian
+    'hu',  # Hungarian
+    'id',  # Indonesian
+    'it',  # Italian
+    'ja',  # Japanese
+    'ka',  # Georgian
+    'kab',  # Kabyle
+    'ko',  # Korean
+    'mk',  # Macedonian
+    'mn',  # Mongolian
+    'nl',  # Dutch
+    'nn-NO',  # Norwegian (Nynorsk)
+    'pl',  # Polish
+    'pt-BR',  # Portuguese (Brazilian)
+    'pt-PT',  # Portuguese (Portugal)
+    'ro',  # Romanian
+    'ru',  # Russian
+    'sk',  # Slovak
+    'sl',  # Slovenian
+    'sq',  # Albanian
+    'sv-SE',  # Swedish
+    'uk',  # Ukrainian
+    'ur',  # Urdu
+    'vi',  # Vietnamese
+    'zh-CN',  # Chinese (Simplified)
+    'zh-TW',  # Chinese (Traditional)
 )
 
+# Bidirectional languages.
+# Locales in here *must* be in `AMO_LANGUAGES` too.
```

Comment:
is it worth an assert?  Otherwise a test

Reply:
Added in 25b842ab92d3a7dbae15cafff7fb68a5ca7dc95b and fixed in cf5e0a674e150ed74e23c91d89bb25e1ea80991b